packages/split-view/src/SplitView.ts

```diff
@@ -0,0 +1,419 @@
+
+/**
+ * @element sp-split-view
+ */
+export class SplitView extends SpectrumElement {
+    public static get styles(): CSSResultArray {
+        return [styles];
+    }
+
+    @property({ type: Boolean, reflect: true })
+    public vertical = false;
+
+    @property({ type: Boolean, reflect: true })
+    public resizable = false;
+
+    @property({ type: Boolean, reflect: true })
+    public collapsible = false;
+
+    /** The minimum size of the primary pane */
+    @property({ type: Number, attribute: 'primary-min' })
+    public primaryMin = 0;
+
+    /** The maximum size of the primary pane */
+    @property({ type: Number, attribute: 'primary-max' })
+    public primaryMax = DEFAULT_MAX_SIZE;
+
+    /** The start size of the primary pane */
+    @property({ type: String, attribute: 'primary-size' })
+    public primarySize?: string;
+
+    /** The minimum size of the secondary pane */
+    @property({ type: Number, attribute: 'secondary-min' })
+    public secondaryMin = 0;
+
+    /** The maximum size of the secondary pane */
+    @property({ type: Number, attribute: 'secondary-max' })
+    public secondaryMax = DEFAULT_MAX_SIZE;
+
+    @property({ type: Number, reflect: true, attribute: 'splitter-pos' })
+    public splitterPos?: number;
+
+    @property({ type: Number, attribute: false })
+    public minPos = 0;
+
+    @property({ type: Number, attribute: false })
+    public maxPos = Infinity;
+
+    @property({ type: Boolean, reflect: true, attribute: 'is-resized-start' })
+    public isResizedStart = false;
+
+    @property({ type: Boolean, reflect: true, attribute: 'is-resized-end' })
+    public isResizedEnd = false;
+
+    @property({ type: Boolean, reflect: true, attribute: 'is-collapsed-start' })
+    public isCollapsedStart = false;
+
+    @property({ type: Boolean, reflect: true, attribute: 'is-collapsed-end' })
+    public isCollapsedEnd = false;
+
+    @property()
+    public label?: string;
+
+    @property({ type: Boolean, attribute: false })
+    private enoughChildren = false;
+
+    @query('slot')
+    private paneSlot!: HTMLSlotElement;
+
+    @query('#splitter')
+    private splitter!: HTMLDivElement;
+
+    private offset = 0;
+
+    private observer?: WithSWCResizeObserver['ResizeObserver'];
+
+    private rect?: DOMRect;
+
+    private _splitterSize?: number;
+
+    public constructor() {
+        super();
+        const RO = ((window as unknown) as WithSWCResizeObserver)
+            .ResizeObserver;
+        if (RO) {
+            this.observer = new RO(() => {
+                this.rect = undefined;
+                this.updateMinMax();
+                this.checkStartEnd();
+            });
+        }
+    }
+
+    public connectedCallback(): void {
+        super.connectedCallback();
+        this.observer?.observe(this);
+    }
+
+    public disconnectedCallback(): void {
+        this.observer?.unobserve(this);
+        super.disconnectedCallback();
+    }
+
+    public get viewSize(): number {
```

Comment:
I think this is where the visual regressions are failing. Try adding the following and running them again: 
```suggestion
    /**
     * @private
     **/
    public get viewSize(): number {
```